client/web/src/cody/subscription/CodySubscriptionPage.tsx

```diff
@@ -93,13 +93,12 @@ export const CodySubscriptionPage: React.FunctionComponent<CodySubscriptionPageP
                         )
                     }
                 >
-                    <PageHeader.Heading as="h2" styleAs="h1">
-                        <div className="d-inline-flex align-items-center">
-                            <CodyColorIcon width={40} height={40} className="mr-2" /> Subscription plans
-                        </div>
+                    <PageHeader.Heading as="h1" className="text-3xl font-medium">
+                        <PageHeaderIcon name="cody-logo" className="mr-3" />
+                        <Text as="span">Subscription plans</Text>
                     </PageHeader.Heading>
                 </PageHeader>
-                <Link to="/cody/manage" className="my-4">
+                <Link to="/cody/manage">
```

Comment:
Out of PR's scope: we should use `CodyProRoutes` enum for route paths where possible. 
```suggestion
                <Link to={CodyProRoutes.Manage}>
```

Reply:
@taras-yemets, this caused problems because I also needed the import, so once I was there, I've added this (admittedly unrelated) commit to let us use `CodyProRoutes` everywhere: https://github.com/sourcegraph/sourcegraph/pull/63380/commits/aec721e124f9a1cbacaf289186e0f330ddab10d2